### PR TITLE
New nova frontend using new refine frontends

### PR DESCRIPTION
**Pull request by aarondfrancis**
Don't merge until Nova 4 is done and the package.json is updated to remove the -dev version. (Need to publish refine-vue2 first)

### CODE REVIEW

**Comment by aarondfrancis** [comment]
Assets are publishing, just need to publish the real refine-vue2 and 3 and reference that and then we'll be good to go

File: nova3/resources/js/components/GroupDivider.vue [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <div class="my-2 text-80 items-center">
+    <div class="border-t w-4 border-60"></div>
+    <div class="mx-2 text-sm">Or</div>
```
Comment (pedro-santiago) [inline nova3/resources/js/components/GroupDivider.vue:4]:
```suggestion
    <div class="mx-2 text-sm">{{ __('Or') }}</div>
```

File: nova3/resources/js/components/OrButton.vue [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<template>
+  <button v-on="$listeners">
+    <heroicon-plus />
+    <span class="pt-px">Or</span>
```
Comment (pedro-santiago) [inline nova3/resources/js/components/OrButton.vue:4]:
```suggestion
    <span class="pt-px">{{ __('Or') }}</span>
```

File: nova4/resources/js/components/Card.vue [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        <div>
+          <query-builder
+            :errors="errors"
+            v-model:blueprint="filter.blueprint"
+            :conditions="filter.conditions"
+            :flavor="flavor"
+          />
+
+          <div class="text-right">
+            <button @click.prevent="collapsed = !collapsed" class="text-sm mr-6 text-80">Collapse</button>
```
Comment (pedro-santiago) [inline nova4/resources/js/components/Card.vue:19]:
```suggestion
            <button @click.prevent="collapsed = !collapsed" class="text-sm mr-6 text-80">{{ __('Collapse') }}</button>
```

File: nova4/resources/js/components/Card.vue [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+            >
+              Filter
+            </button>
+          </div>
+        </div>
+      </slide-down>
+      <slide-down :show="collapsed">
+        <div class="border rounded-lg shadow border-50 p-4 text-80 bg-white flex items-center justify-between text-sm">
+          <div>{{ collapsedText }}</div>
+          <button class="text-80" @click.prevent="collapsed = !collapsed">Expand Filter</button>
```
Comment (pedro-santiago) [inline nova4/resources/js/components/Card.vue:32]:
```suggestion
          <button class="text-80" @click.prevent="collapsed = !collapsed">{{ __('Expand Filter') }}</button>
```

File: nova4/resources/js/components/GroupDivider.vue [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <div class="my-2 text-gray-400 items-center">
+    <div class="border-t w-4 dark:border-gray-800"></div>
+    <div class="mx-2 text-xs font-bold">Or</div>
```
Comment (pedro-santiago) [inline nova4/resources/js/components/GroupDivider.vue:4]:
```suggestion
    <div class="mx-2 text-xs font-bold">{{ __('Or') }}</div>
```

File: nova4/resources/js/components/OrButton.vue [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<template>
+  <button>
+    <heroicon-plus />
+    <div class="pt-px text-xs font-bold">Or</div>
```
Comment (pedro-santiago) [inline nova4/resources/js/components/OrButton.vue:4]:
```suggestion
    <div class="pt-px text-xs font-bold">{{ __('Or') }}</div>
```

File: nova4/resources/js/components/Card.vue [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+    }
+  },
+
+  computed: {
+    refineParameterName() {
+      return `${this.resourceName}_refine`;
+    },
+
+    collapsedText() {
+      return this.calculateCollapsedText(this.lastAppliedBlueprint);
```
Comment (pedro-santiago) [inline nova4/resources/js/components/Card.vue:127]:
```suggestion
      return this.__(this.calculateCollapsedText(this.lastAppliedBlueprint));
```

**Review by pedro-santiago — COMMENTED** [review]
Code looks really really awesome as always! Added a few translations to make it a bit more translatable :)

**Comment by aarondfrancis** [comment]
@pedro-santiago oh nice, thank you so much for that!

**Pull request merged**